install/main.yml

```diff
@@ -99,6 +63,9 @@
     - role: roundcube
       when: webmail.install and webmail.type == 'roundcube'
       tags: roundcube
+    - role: imapproxy
+      when: sogo.install
```

Comment:
`or roundcube.install` also, right?
The tags should include sogo and roundcube also I think.